contrib/expose-gitlab-ssh-port.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -ev
+
+if ! id -u git >> /dev/null 2>&1; then
+  groupadd -g 1010 git
```

Comment:
I think we should make the group id changeable via variable or a flag 
```
GITLAB_USERGROUP=$1
GITLAB_USERGROUP=${GITLAB_USERGROUP:-1010}
```

Reply:
👍